src/backend/common/cache_clearing/get_affected_queries.py

```diff
@@ -0,0 +1,313 @@
+from typing import Any, List, Set, Tuple, Type
+
+from google.cloud import ndb
+
+from backend.common.models.cached_model import TAffectedReferences
+from backend.common.models.district_team import DistrictTeam
+from backend.common.models.event import Event
+from backend.common.models.event_team import EventTeam
+from backend.common.models.keys import TeamKey
+from backend.common.queries import (
+    award_query,
+    district_query,
+    event_details_query,
+    event_query,
+    match_query,
+    media_query,
+    robot_query,
+    team_query,
+)
+from backend.common.queries.database_query import CachedDatabaseQuery
+
+TCacheKeyAndQuery = Tuple[str, Type[CachedDatabaseQuery]]
+
+
+def _queries_to_cache_keys_and_queries(
+    queries: List[CachedDatabaseQuery],
+) -> List[TCacheKeyAndQuery]:
+    out = []
+    for query in queries:
+        out.append((query.cache_key, type(query)))
+    return out
+
+
+def _get_team_page_num(team_key: TeamKey) -> int:
+    return int(team_key[3:]) // team_query.TeamListQuery.PAGE_SIZE
```

Comment:
This looks like it's just moving over code but - can we DRY this? It looks like this same bit of code is used in `team_query.py`

Reply:
Good catch :+1: